src/main/perl/FileEditor.pm

```diff
@@ -58,13 +66,45 @@ sub _is_valid_source
     return -f $fn;
 }
 
+sub _is_reference_newer
```

Comment:
This is the usual stuff in CAF::Filexxx. There are already quite a lot of it around. I can add a comment that you should look at `CAF::FileWriter` for details. I can add a POD but this is a very internal method and as there were none for `_is_valid` I decided to do the same...